stack.yaml

```diff
@@ -14,6 +14,7 @@ packages:
 - grpc/client
 - grpc/server
 - graphql
+- servant/server
```

Comment:
Please add this line also to `stack-14.yaml` and `stack-nightly.yaml`.

Reply:
Done!